test/fixtures_hub.js

```diff
@@ -0,0 +1,806 @@
+
+module.exports = function() {
+    return [
+
+        // 0:
+        {
+            "request": {
+                "url": 'http://hub.localhost:8001/api/v1/change/',
+                "method": 'POST',
+                "data": {
+                    "registrant_id": "cb245673-aa41-4302-ac47-00000000001",
+                    "action": "pmtct_nonloss_optout",
+                    "data": {
+                        "reason": "unknown"
+                    }
+                }
+            },
+            "response": {}
+        },
+
+        // 1:
+        {
+            'request': {
+                'method': 'GET',
+                'params': {
+                    "deprecated": "fixture"
+                },
+                'headers': {
```

Comment:
@imsickofmaps js fixtures don't check the headers, so I like to omit the headers so you don't think you're checking them when you're not - do you agree with that?